frame/staking/src/benchmarking.rs

```diff
@@ -189,6 +189,7 @@ impl<T: Config> ListScenario<T> {
 		)?;
 
 		// find a destination weight that will trigger the worst case scenario
+		// TODO: This is a hack, might be great to implement this differently.
```

Comment:
if this remains, it needs to be mentioned in an issue. We might already have an issue to simplify `ListScenario`.